x-pack/filebeat/module/suricata/README.md

```diff
@@ -0,0 +1,60 @@
+# Suricata module
+
+Shove the content of this repo under `filebeats/module/suricata` for testing,
```

Comment:
I'll clean this up along with the documentation in a separate PR.

Reply:
Yeah I put this file in there as a convenience for the handoff. I tend to think that having a bit of a readme this deep down in the module can be helpful to people who will collaborate on this in the future, while not being in the way otherwise. I would leave it in there after a cleanup. I'd probably remove the whole "Caveats" section but leave the getting started tips.